composer.json

```diff
@@ -64,11 +64,11 @@
     },
     "extra": {
         "installer-paths": {
-            "web/core": ["type:drupal-core"],
-            "web/libraries/{$name}": ["type:drupal-library"],
-            "web/modules/contrib/{$name}": ["type:drupal-module"],
-            "web/profiles/contrib/{$name}": ["type:drupal-profile"],
-            "web/themes/contrib/{$name}": ["type:drupal-theme"],
+            "drupal/core": ["type:drupal-core"],
```

Comment:
change this to match /webroot

Reply:
command to update just the lock file based on new paths `composer update --lock`